src/main/java/podo/odeego/domain/group/service/GroupQueryService.java

```diff
@@ -68,4 +69,16 @@ public void verifyGroupExists(UUID groupId) {
 			throw new GroupNotFoundException("Cannot find group for groupId=%s".formatted(groupId.toString()));
 		}
 	}
+
+	public GroupMemberExistResponse isSubmitted(UUID groupId, Long memberId) {
+		Group group = groupRepository.findFetchById(groupId)
+			.orElseThrow(
+				() -> new GroupNotFoundException("Cannot find group for groupId=%s".formatted(groupId.toString())));
+		
+		return new GroupMemberExistResponse(
+			group.groupMembers()
+				.stream()
+				.anyMatch(groupMember -> groupMember.getMemberId().equals(memberId))
```

Comment:
a: Group에 속한 참가자가 출발지를 제출했는지를 확인하는 stream 로직이 Service에 있는 것 보단 도메인 객체에 있는게 더 나을 것 같은데 어떻게 생각하시나요? 
Group이 GroupMember를 순회하면서 memberId가 일치하는지 확인하는 작업은 구체적인 사항 같습니다!

```suggestion
		group.isGroupMemberSubmitted(memberId);
```

Reply:
group 안으로 메서드를 넣으면 좋을 것 같네요! 감사합니다!